sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/aad/AADAuthenticationProperties.java

```diff
@@ -299,4 +311,22 @@ public boolean isAllowedGroup(String group) {
                        .orElseGet(Collections::emptyList)
                        .contains(group);
     }
+
+    @Override
+    public void afterPropertiesSet() throws Exception {
+        if (!StringUtils.hasText(graphMembershipUri)) {
+            graphMembershipUri = graphBaseUri + "v1.0/me/memberOf";
+        }
+        validate();
+    }
+
+    private String addSlash(String uri) {
+        return uri.endsWith("/") ? uri : uri + "/";
+    }
+
+    private void validate() {
+        if (!graphMembershipUri.startsWith(graphBaseUri)) {
+            throw new IllegalStateException("The domains of graphBaseUri and graphMembershipUri are inconsistent.");
```

Comment:
`azure.activedirectory.graph-base-uri should be the prefix of azure.activedirectory.graph-membership-uri. azure.activedirectory.graph-base-uri = blabla, azure.activedirectory.graph-membership-uri = blabla.`